### PR TITLE
Hello React Frontend Submission

### PR DESCRIPTION
**Pull request by fmanimashaun**
Create a simple react application that consume API from a rails API only backend

[Link to backend PR](https://github.com/fmanimashaun/hello-rails-back-end/pull/2)

### CODE REVIEW

**Review by OyePriscilla — COMMENTED** [review]
Hi @fmanimashaun 👍 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

**You have submitted a project, that has no linters installed.** 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Greeting from './features/greeting/Greeting';
+
+function App() {
+  return (
+    <div>
+      <Greeting />
+    </div>
+  );
+}
```
Comment (mckent05) [inline src/App.js]: - You have done well so far, but according to the project requirements, your app component should have a route that leads to the Greeting component. Please wrap your Greeting component in a route as shown below 👍 
```javascript
   function App() {
     return (
       <div className="App">
         <Routes>
           <Route path="/" element={<Greeting />} />
         </Routes>
       </div>
  );
}
```
reqewraewrawe

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @fmanimashaun, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍
- Good job setting up the frontend for your application 👍 👍  


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (fmanimashaun) [reply src/App.js]: resolved

**Review by mckent05 — APPROVED** [review]
Hi @fmanimashaun , 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**